Integrations/integration-fireeye.yml

```diff
@@ -28,7 +27,7 @@ configuration:
   defaultvalue: 1.1.0
   type: 0
   required: false
-- display: Trust any certificate (unsecure)
+- display: Do not validate server certificate (insecure)
```

Comment:
Please return `display:` to "Trust any certificate (unsecure)"

Reply:
Changed display name of field to it default.